lib-driver/caqti_driver_mariadb.mli

```diff
@@ -12,14 +12,16 @@
  *
  * You should have received a copy of the GNU Lesser General Public License
  * along with this library.  If not, see <http://www.gnu.org/licenses/>.
- *)
+*)
 
 (** Caqti driver for MariaDB (bindings).
 
-    This driver is implement in terms of the mariadb OPAM package.
+    This driver is implemented in terms of the mariadb OPAM package.
     It handles URIs of the form
 
-      {[mariadb://<user>:<password>@<host>:<port>/?socket=<socket>]}
+    {[
+      mariadb://<user>:<password>@<host>:<port>/?socket=<socket>
+    ]}
```

Comment:
Yes, maybe that looks better in the code, but I suggest also removing empty lines above and below.

Reply:
Fixed for all of the driver.mli files